crates/derive/src/combiner.rs

```diff
@@ -65,6 +65,10 @@ impl Combiner {
         }
     }
 
+    pub fn doc_count(&self) -> usize {
```

Comment:
nit: `len` does seem an appropriate name here ? Since a combiner is-a collection